### ISSUE TITLE
Not UWP compatible

### ISSUE THREAD

**Issue by bc3tech** [mattleibow/Microsoft-Band-SDK-Bindings#12]
```
Restoring packages for 'App'.
Restoring packages for C:\Users\...\Windows\UWP\App\project.json...
Xamarin.Microsoft.Band.Native 1.3.8 is not compatible with UAP,Version=v10.0.
Some packages are not compatible with UAP,Version=v10.0.
Xamarin.Microsoft.Band.Native 1.3.8 is not compatible with UAP,Version=v10.0 (win10-arm).
Some packages are not compatible with UAP,Version=v10.0 (win10-arm).
Xamarin.Microsoft.Band.Native 1.3.8 is not compatible with UAP,Version=v10.0 (win10-arm-aot).
Some packages are not compatible with UAP,Version=v10.0 (win10-arm-aot).
Xamarin.Microsoft.Band.Native 1.3.8 is not compatible with UAP,Version=v10.0 (win10-x86).
Some packages are not compatible with UAP,Version=v10.0 (win10-x86).
Xamarin.Microsoft.Band.Native 1.3.8 is not compatible with UAP,Version=v10.0 (win10-x86-aot).
Some packages are not compatible with UAP,Version=v10.0 (win10-x86-aot).
Xamarin.Microsoft.Band.Native 1.3.8 is not compatible with UAP,Version=v10.0 (win10-x64).
Some packages are not compatible with UAP,Version=v10.0 (win10-x64).
Xamarin.Microsoft.Band.Native 1.3.8 is not compatible with UAP,Version=v10.0 (win10-x64-aot).
Some packages are not compatible with UAP,Version=v10.0 (win10-x64-aot).
Package restore failed for 'App'.
Package restore failed. Rolling back package changes for 'App'.
```

I'll work on this, unless you plan to.

**Comment by bc3tech** [comment]
from https://oren.codes/2015/08/29/uwp-nuget-package-dependencies/:

> For a UWP Class Library package, you should have the following in your NuSpec:
>     •A dependency group with the uap10.0 TFM
>     •In your Project Build options for Release mode, choose “generate library layout”
>     •Copy the entire directory structure of the output to your \lib\uap10.0 dir.

**Comment by mogwai** [comment]
When will the  [nuget package](https://components.xamarin.com/view/microsoft-band-sdk) be appliable to a project?

**Comment by bc3tech** [comment]
@mattleibow please merge ASAP & release a nupkg update. Tested and fixes the issue w/ .Native.nupkg not applying properly to UWP and other .Net projects where you desired **Microsoft's** Band SDK to be pulled in

**Comment by mattleibow** [comment]
@bc3tech  I made a few comments, please have a look. I was away this weekend, so only got to it now.

**Comment by mattleibow** [comment]
Fixed in commit 7625839d5de49846879204d0f926b56ef14a9c44.
The problem was two-fold, the folder name should be `uap10.0` with a period, and the `Readme.md` does not represent an empty folder, so `_._` was used instead.

**Comment by mattleibow** [comment]
@bc3tech There should be an update on NuGet: v1.3.8.1
